os/android/drmhwctwo.cpp

```diff
@@ -211,6 +211,11 @@ HWC2::Error DrmHwcTwo::HwcDisplay::RegisterVsyncCallback(
 HWC2::Error DrmHwcTwo::HwcDisplay::AcceptDisplayChanges() {
   supported(__func__);
   uint32_t num_changes = 0;
+  if (!checkValidateDisplay) {
```

Comment:
We should reset this to false in this call, otherwise we check this only the first time and if its set to true this check doesnt add any value for further AcceptDisplayChanges.